### PR TITLE
chore(endpoints): add new user service endpoint

### PR DESCRIPTION
**Pull request by gregdeane**
#39 

@cjhowald @evdokim @sherifkandeel 

I confirmed with Sherif that the url is correct up from DEV to STAGING. However, we need to lock down the final PROD url soon so we can release this version. Sherif, Kim / Please reply with the final prod URL.

Also, the property name `USER_SERVICE_NEW` is pretty lame. We should come up with a better one. Any suggestions?

### CODE REVIEW

File: src/environments/environments.constant.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ module.exports.ENVIRONMENTS = {
     TOKEN_SERVICE: 'https://token-management.norris.zalan.do',
     MERCHANT_SERVICE: 'https://merchant-management.norris.zalan.do',
     GOODDATA_SERVICE: 'https://gooddata.norris.zalan.do',
-    MODULE_SERVICE: 'https://module-service.norris.zalan.do'
+    MODULE_SERVICE: 'https://module-service.norris.zalan.do',
+    USER_SERVICE_NEW: 'https://user-service.norris.zalan.do'
```
Comment (sherifkandeel) [inline src/environments/environments.constant.js]: `user-service` seems a good name in production for me. pending others approval

**Comment by cjhowald** [comment]
Revert `dist`changes please

**Comment by cjhowald** [comment]
I think `USER_SERVICE_NEW` is fine for now. When will we be able to phase everything over to the new endpoints? At that point, we can shift it to `USER_SERVICE` and retire the `NEW`.

File: src/environments/environments.constant.js (hunk 1 above)
Comment (gregdeane) [inline src/environments/environments.constant.js]: @evdokim Is `https://user-service.norris.zalan.do` okay for the PROD endpoint?

**Comment by gregdeane** [comment]
@cjhowald Removed `dist`

**Comment by gregdeane** [comment]
@cjhowald Not sure when we can shift everything over to the new endpoint but will definitely rename property when we do.

File: src/environments/environments.constant.js (hunk 1 above)
Comment (evdokim) [inline src/environments/environments.constant.js]: @gregdeane yes, since user-service.norris.zalan.do it is the best option.

**Comment by cjhowald** [comment]
👍

**Comment by gregdeane** [comment]
👍

**Pull request merged**